app/src/components/market/market_bond/market_bond.tsx

```diff
@@ -272,6 +272,12 @@ const MarketBondWrapper: React.FC<Props> = (props: Props) => {
               value={`${bigNumberToString(bondNativeAssetAmount, nativeAsset.decimals)} ${symbol}`}
             />
             <TransactionDetailsLine />
+            <TransactionDetailsRow
+              state={ValueStates.important}
```

Comment:
![Screenshot 2021-08-12 at 14 05 40](https://user-images.githubusercontent.com/33226956/129193830-c67649f3-ff70-402a-aa5e-fe49d8db3f69.png)
I think because you put important here it shows value in different color then the rest....